src/components/UI/SearchBar/index.css

```diff
@@ -1,6 +1,8 @@
 .SearchBar {
   grid-area: search;
   width: 100%;
+  display: inline-block;
```

Comment:
is this display: inline-block needed?

Reply:
No, must've been a leftover from my testing, I'll fix that in an upcoming commit